hawc/apps/assessment/templates/assessment/effecttag_form.html

```diff
@@ -16,7 +16,7 @@
   <script type="text/javascript">
     $(document).ready(function(){
       $('#id_name').on('keyup', function(){
-        $('#id_slug').val(window.app.HAWCUtils.urlify($(this).val(), 50));
+        $('#id_slug').val(window.app.HAWCUtils.urlify($(this).val(),128));
```

Comment:
should the default length be 50? https://docs.djangoproject.com/en/3.2/ref/models/fields/#slugfield

Reply:
doh; i think you did do that, and i'm looking at a different case. my bad.